core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -85,8 +102,9 @@ protected function configure(): void
     {
         $this
             ->addOption('time-limit', 'l', InputOption::VALUE_OPTIONAL, 'Time limit in seconds', '0')
-            ->addOption('concurrent', 'c', InputOption::VALUE_OPTIONAL, 'Run multiple processes concurrently', '1')
-            ->addOption('throttle', 't', InputOption::VALUE_OPTIONAL, 'Pause between resizes to limit CPU utilization, 0.1 relates to 10% CPU usage', '1')
+            ->addOption('concurrent', 'c', InputOption::VALUE_OPTIONAL, 'Run multiple processes concurrently with a value larger than 1 or pause between resizes to limit CPU utilization with values lower than 1.0', '1')
```

Comment:
Would it make sense to increase the default value? Or is a higher parallelization already a problem on most hostings?

Reply:
I think so, yes. On most hosting packages you should probably use a value less than `0.5`.